iceberg/iceberg-handler/src/test/results/positive/iceberg_insert_overwrite_partition.q.out

```diff
@@ -487,32 +488,32 @@ STAGE PLANS:
             Map Operator Tree:
                 TableScan
                   alias: ice_parquet_string
-                  Statistics: Num rows: 4 Data size: 1084 Basic stats: COMPLETE Column stats: COMPLETE
+                  Statistics: Num rows: 4 Data size: 2224 Basic stats: COMPLETE Column stats: NONE
                   Select Operator
                     expressions: name (type: string), age (type: int), country (type: string), state (type: string)
                     outputColumnNames: _col0, _col1, _col2, _col3
-                    Statistics: Num rows: 4 Data size: 1084 Basic stats: COMPLETE Column stats: COMPLETE
+                    Statistics: Num rows: 4 Data size: 2224 Basic stats: COMPLETE Column stats: NONE
                     Reduce Output Operator
                       key expressions: _col2 (type: string), _col3 (type: string)
                       null sort order: aa
                       sort order: ++
                       Map-reduce partition columns: _col2 (type: string), _col3 (type: string)
-                      Statistics: Num rows: 4 Data size: 1084 Basic stats: COMPLETE Column stats: COMPLETE
+                      Statistics: Num rows: 4 Data size: 2224 Basic stats: COMPLETE Column stats: NONE
                       value expressions: _col0 (type: string), _col1 (type: int), _col2 (type: string), _col3 (type: string)
                     Select Operator
                       expressions: _col0 (type: string), _col1 (type: int), _col2 (type: string), _col3 (type: string)
                       outputColumnNames: name, age, country, state
-                      Statistics: Num rows: 4 Data size: 1084 Basic stats: COMPLETE Column stats: COMPLETE
+                      Statistics: Num rows: 4 Data size: 2224 Basic stats: COMPLETE Column stats: NONE
```

Comment:
many occurrences of COMPLETE -> NONE, is it a statistics regression?

Reply:
no, stats was completely wrong

Reply:
okay, I'm assuming those can still be collected with ANALYZE TABLE ... FOR COLUMNS if we want to have them (it's just not the scope of this ticket)

Reply:
yes, there is a q test for that - `puffin_col_stats.q`

Reply:
@abstractdog, sorry my bad. 
Now, since we no longer consider table as unpartitioned, we need to calculate, store and fetch col stats per partition (same as for basic stats implemented here). Table col stats can't be used anymore.
I'll create a follow-up ticket for that since it's a big change.

Reply:
added fallback to table column stats in https://github.com/apache/hive/pull/5498/commits/8f24713a3d2e303dac5fc8df1c7929e26e7dc561